pmd-core/src/main/java/net/sourceforge/pmd/properties/PropertyFactory.java

```diff
@@ -162,13 +162,32 @@ public static RegexPropertyBuilder regexProperty(String name) {
     }
 
 
+    /**
+     * Returns a builder for a string property. The property descriptor
+     * will accept any string, and performs no expansion of escape
+     * sequences (e.g. {@code \n} in the XML will be represented as the
+     * character sequence '\' 'n' and not the line-feed character '\n'). This
+     * behaviour could be changed with PMD 7.0.0.
+     *
+     * @param name Name of the property to build
+     *
+     * @return A new builder
+     */
     public static GenericPropertyBuilder<String> stringProperty(String name) {
         return new GenericPropertyBuilder<>(name, ValueParserConstants.STRING_PARSER, String.class);
     }
 
 
+    /**
+     * Returns a builder for a property having as value a list of strings. The
+     * format of the individual items is the same as for {@linkplain #stringProperty(String) stringProperty}.
+     *
+     * @param name Name of the property to build
+     *
+     * @return A new builder
+     */
     public static GenericCollectionPropertyBuilder<String, List<String>> stringListProperty(String name) {
-        return stringProperty(name).toList().delim(MultiValuePropertyDescriptor.DEFAULT_DELIMITER);
```

Comment:
I think, I've added this delimiter (also for the other list types). Is it already safe to remove it? This requires, that every stringListProperty-callee defines its own delimiter...
I think, I noticed that it was missing, when the documentation has been generated and output "\0" as the delimiter for some properties.

Reply:
I added a comment on PropertyBuilder addressing that

Reply:
Ah, I missed that change :smile: :+1: